packages/gatsby-transformer-javascript-static-exports/README.md

```diff
@@ -0,0 +1,91 @@
+# gatsby-transformer-js
+
+Parses js files.
+
+## Install
+
+`npm install --save gatsby-transformer-js`
+
+## How to use
+
+```javascript
+// In your gatsby-config.js
+plugins: [
+  `gatsby-transformer-js`,
+]
+```
+
+## Parsing algorithm
+
+The algorithm for uses babylon and traverse (from the babel family of code) to statically read the data exports.
+
+In a .js|.jsx file, export a data object to set your metadata variables, like so:
+```javascript
+import React from 'react'
+
+exports.data = {
+  title: 'This is a title',
+}
+
+export default MyComponent ...
+```
+
+You can also use a named export for the data object:
+
+```javascript
+export const data = {
+  title: 'This is a title',
+}
+```
+
+## How to query
+
+You'd be able to query your letters like:
+
+```graphql
+{
+  allJsFrontmatter {
+    edges {
+      node {
+        data {
+          JSFrontmatter
```

Comment:
It'd be more clear if the fields here matched the fields of the sample export above. So either add more fields above or remove the extra fields here.